templates/ovncontroller/ovn.sh

```diff
@@ -13,6 +13,9 @@ else
     # ovn-appctl is not present. Use openvswitch run dir path.
     OVNCTL_DIR=openvswitch
 fi
-exec ovn-controller unix:/var/run/openvswitch/db.sock -vfile:off \
+
+HOSTNAME=`/usr/bin/hostname 2>/dev/null`
```

Comment:
FWIW the 2>/dev/null is redundant here as it won't write anything to stderr, and if it did that wouldn't end up in the HOSTNAME variable. A more concise version would be:

HOSTNAME=$(hostname)

You could also pass it from the pod with the downward api:

      env:
        - name: HOSTNAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name

https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#the-downward-api